### PR TITLE
Queryset pagination order and verbosity

### PR DESCRIPTION
**Pull request by DannyAziz**
Added `.order_by('id')` as was getting inconsistent results after rebuilding the index as shown in #151 

Verbosity isn't important but I liked the feature in 'django-haystack'

### CODE REVIEW

File: django_elasticsearch_dsl/documents.py [hunk 1]
```diff
@@ -206,7 +207,12 @@ def _get_actions(self, object_list, action):
             paginator = Paginator(
                 object_list, self._doc_type.queryset_pagination
             )
+            number_of_pages = len(paginator.page_range)
+            current_page = 0
             for page in paginator.page_range:
+                current_page += 1
+                print("NUMBER OF PAGES: " + str(number_of_pages))
```
Comment (crazyscientist) [inline django_elasticsearch_dsl/documents.py]: If I may offer some criticism: Maybe you would like to remove the `print` statements before this gets merged?

**Comment by DannyAziz** [comment]
@crazyscientist Are there any other changes that I should make? I would love to see this merged

**Comment by crazyscientist** [comment]
@DannyAziz To me the PR looks good now, but I have no permission to merge/accept.

@sabricot What do you think?

**Comment by ntcong** [comment]
@safwanrahman can you take a look at this? order_by is really important when you have a big dataset. Adding order_by can be a good immediate step before a better approach, like https://github.com/sabricot/django-elasticsearch-dsl/pull/102

**Comment by safwanrahman** [comment]
If the pk is not incremental integer, will this work? I have doubt that in the case the primary key is not integer, like uuid, it will work actually!

**Comment by safwanrahman** [comment]
@DannyAziz Can you rebase it upon master so its possible to merge?

**Comment by safwanrahman** [comment]
Thanks @ntcong for mentioning me here

**Comment by DannyAziz** [comment]
@safwanrahman I will rebase soon 👍 

If you look at these 2 lines:
```
primary_key_field_name = self._doc_type.model._meta.pk.name
return self._doc_type.model._default_manager.all().order_by(primary_key_field_name)
```
It gets the name of the primary key for that model instead of assuming you are using PK, I am using UUID's in my project and it works fine

**Comment by DannyAziz** [comment]
@safwanrahman I've rebased the fork to master would love to see this merged 👍

**Comment by DannyAziz** [comment]
My bad this needs some fixing, hold on...

**Comment by DannyAziz** [comment]
@safwanrahman The tests that are failing don't seem to be related to my changes, any help?

**Comment by safwanrahman** [comment]
This looks good. r+ from my end

**Pull request merged**